lib/utils/slices/slices.go

```diff
@@ -36,3 +36,22 @@ func FilterMapUnique[T any, S cmp.Ordered](ts []T, fn func(T) (s S, include bool
 	slices.Sort(ss)
 	return slices.Compact(ss)
 }
+
+func SliceToPointerSlice[T any](arr []T) []*T {
```

Comment:
Nit: Perhaps `arr`-> `in` to match `out`.

Reply:
Much more pleasant on the eyes